Cargo.toml

```diff
@@ -71,3 +71,7 @@ name = "blinky"
 
 [[example]]
 name = "orientation"
+
+[[example]]
+name = "buttons"
+# required-features = ["unflappable"]
```

Comment:
```suggestion
```
I'm assuming this was an erroneous copy-paste, would you mind removing it please?

Reply:
Done.